ui/src/pages/overview/overview-page.module.scss

```
@@ -0,0 +1,7 @@
+.root {
```

nit: Can we pick a better name for this class?

```suggestion
.destinations-paper {
```